component/src/main/ballerina/mongodb/actions.bal

```diff
@@ -14,72 +14,57 @@
 // specific language governing permissions and limitations
 // under the License.
 
-documentation {
-    The Caller Actions for MongoDB databases.
-}
+# The Caller Actions for MongoDB databases.
 public type CallerActions object {
-    documentation {
-        The find operation implementation which selects a document in a given collection.
 
-        P{{collectionName}} The name of the collection to be queried
-        P{{queryString}} Query to use to select data
-        R{{}} `json` result from the find operation or `error` if an error occurs
-    }
+    # The find operation implementation which selects a document in a given collection.
+
+    # + collectionName - The name of the collection to be queried
+    # + queryString - Query to use to select data
+    # + return - `json` result from the find operation or `error` if an error occurs
     public extern function find(string collectionName, json? queryString) returns (json|error);
 
-    documentation {
-        The findOne operation implementation which selects the first document match with the query.
+    # The findOne operation implementation which selects the first document match with the query.
```

Comment:
Shall we change the description to "the first document `that matches` with the query"

Reply:
Fixed in 8a667b927e85d5481090f39f476d26c921626a4f